MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+</table> 
+
+<table>
+<tr>
+<th>Xamarin</th>
+</tr>
+<tr>
+<td>
+{% tabs %} 
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+...     
+<progressBar:SfCircularProgressBar IndicatorInnerRadius="0.9"
+                                   IndicatorOuterRadius="1"
+                                   TrackInnerRadius="0.9"
+                                   TrackOuterRadius="1"
+                                   Progress="50">
+
+</progressBar:SfCircularProgressBar>
+
+...
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+
+...
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.IndicatorInnerRadius = 0.9;
+circularProgressBar.IndicatorOuterRadius = 1;
+circularProgressBar.TrackInnerRadius = 0.9;
+circularProgressBar.TrackOuterRadius = 1;
+circularProgressBar.Progress = 50;
+this.Content = circularProgressBar;
+
+...
+
+{% endhighlight %}
+{% endtabs %}
+</td>
+</tr>
+<tr>
+<th>.NET MAUI</th>
+</tr>
+<tr>
+<td>
+{% tabs %} 
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+...
+<progressBar:SfCircularProgressBar ProgressThickness="10"
+                                   ProgressRadiusFactor="1"
+                                   TrackThickness="10"
+                                   TrackRadiusFactor="1"
+                                   ProgressCornerStyle="BothCurve"
+                                   TrackCornerStyle="BothCurve"
+                                   ThicknessUnit="Pixel"
+                                   Progress="50">
+    
+</progressBar:SfCircularProgressBar>
+...
+
+</ContentPage>
+ 
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.ProgressThickness = 10;
+circularProgressBar.ProgressRadiusFactor = 1;
+circularProgressBar.TrackThickness = 10;
+circularProgressBar.TrackRadiusFactor = 1;
+circularProgressBar.ProgressCornerStyle = CornerStyle.BothCurve;
+circularProgressBar.TrackCornerStyle = CornerStyle.BothCurve;
+circularProgressBar.ThicknessUnit = SizeUnit.Pixel;
+circularProgressBar.Progress = 50;
+this.Content = circularProgressBar;
+...
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+</tr>
+</table>
+
+## Unsupported features from Xamarin.Forms
+
+* [`ShowProgressValue`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue) support has not been provided in the SfCircularProgressBar. Instead, you can achieve it using the `Content` property. Please visit Custom Content page for more details about the `Content` property.
```

Comment:
* [`ShowProgressValue`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfCircularProgressBar.html#Syncfusion_XForms_ProgressBar_SfCircularProgressBar_ShowProgressValue) support has not been provided in the SfCircularProgressBar. Instead, you can achieve it using the `Content` property. Please visit the Custom Content page for more details about the `Content` property.

Reply:
done